src/ws_server.cu

```diff
@@ -165,20 +171,17 @@ class AprilTagHandler : public seasocks::WebSocket::Handler {
 
     return true;
   }
-
-  // TODO: Implement a faster version of this method!
-  void rotateImage(cv::Mat* bgr_img, const float angle) {
-    cv::Point2f center((bgr_img->cols - 1) / 2.0, (bgr_img->rows - 1) / 2.0);
-    cv::Mat matRotation = cv::getRotationMatrix2D(center, angle, 1.0);
-    cv::Mat rotatedImage;
-    cv::warpAffine(*bgr_img, rotatedImage, matRotation, bgr_img->size());
-    *bgr_img = rotatedImage.clone();
+//Flipcode -1 = both directions
+  void flipVertical(cv::Mat* bgr_img){
+    cv::flip(*bgr_img, *bgr_img, 0);
```

Comment:
I tested the code and when I click flipVertical or flipHorizonal checkboxes the apriltag stops being detected.  I think the problem might be this line, where you are giving the input and the output to be the same.  Can you try modifying it like this:

```
void flipVertical(const cv::Mat& bgr_img, cv::Mat* flipped_img){
    cv::flip(bgr_img, *flipped_img, 0);
}
```

I'm wondering if it's getting screwed up because the input and output are the same.

Reply:
Fixed in [b7df4bb](https://github.com/cgpadwick/apriltags_cuda/pull/51/commits/b7df4bb2b4d1b8d12953b7dbca72f60d7ee90f77)